spec/spec_helper.rb

```diff
@@ -4,8 +4,8 @@
 require 'rubygems'
 require 'rspec'
 require 'pp' # See https://github.com/fakefs/fakefs/issues/99
-require 'pry-byebug'
 require 'fakefs/spec_helpers'
+require 'zlib'
```

Comment:
@maeve It was throwing an error that it could't find zlib when running specs, so I did this. Not sure why it didn't do it before? Maybe the ruby bump, although unlikely.